src/components/pages/Contacts/index.tsx

```diff
@@ -234,6 +246,51 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
           </Modal.Footer>
         </Modal>
 
+        <Modal show={showSendModal}>
+          <Modal.Body>
+            <p>
+              {' '}
+              Send a message to {contactBeingSentMail?.first_name}{' '}
+              {contactBeingSentMail?.last_name}
+            </p>
+            <Form className="d-flex flex-column">
+              <Form.Group controlId="formDocdrop">
+                <Form.Label>Attach file</Form.Label>
+                <Docdrop
+                  uploadFile={(file) => setUploadedFile(file)}
+                  uploadedFile={uploadedFile}
+                  acceptedFormat="application/pdf/*"
+                  acceptedFormatLabel="PDF"
+                />
+              </Form.Group>
+            </Form>
+          </Modal.Body>
+          <Modal.Footer>
+            <Button
+              onClick={() => {
+                setShowSendModal(false);
+                setContactBeingSentMail(null);
+              }}>
+              Close
+            </Button>
+            <Button
+              onClick={() => {
+                sendDirectMessage(token, {
+                  contact_id: contactBeingSentMail
```

Comment:
^if we just use selectedContact, we wont need this ccheck

Reply:
mmm correct me if i'm wrong but I think we still need the check because JS interpreter doesn't know that this method couldn't be called unless the modal was being shown?